backend/tests/events.test.ts

```diff
@@ -92,3 +93,127 @@ describe ("Testing DELETE event",() => {
          expect(response.status).toBe(500);
        })
       })
+
+
+describe ("Testing GET event by eventID", () => {
+  test("Get existing event", async () => {
+
+    const events = await request(app).get("/events");
+    const eventid = events.body[1].id;
+    const response = await request(app).get("/events/" + eventid);
+    expect(response.status).toBe(200);
+  })
+
+  test("Get non-existing event", async () => {
+
+    const eventid = -1;
+    const response = await request(app).get("/events/" + eventid);
+    expect(response.body).toBe(null);
+  })
+})
+
+describe ("Testing GET all attendees", () => {
+  test("Get attendees for existing event", async () => {
+
+    const events = await request(app).get("/events");
+    const eventid = events.body[0].id;
+    const response = await request(app).get("/events/" + eventid + "/attendees");
+    expect(response.status).toBe(200);
+  })
+})
+
+  test("Get attendees for invalid event", async () => {
+
+    const eventid = -1;
+    const response = await request(app).get("/events/" + eventid + "/attendees");
+    expect(response.body.length).toBe(0);
+  })
+
+  describe ("Testing POST/events/:eventid/:attendeeid", () => {
+    test("Add attendee for existing event", async () => {
+
+      const events = await request(app).get("/events");
+      const users = await request(app).get("/users");
+      const eventID = events.body[1].id;
+      const attendeeid = users.body[1].id;
+      const response = await request(app).post("/events/" + eventID + "/"+ attendeeid);
+      expect(response.status).toBe(200);
```

Comment:
You can be a little more clever with this test. If you add an attendee to an event, why don't you check if it was successful by seeing if that user now exists in the list of attendees for that event? 
Let me know if you have questions